IO/src/av/CMakeLists.txt

```diff
@@ -58,15 +58,12 @@ function(setup_av_plugin PLUGIN_NAME BACKEND_NAME USE_CUDA)
 
   set_property(TARGET ${PLUGIN_NAME} PROPERTY CXX_STANDARD 14)
 
-  set(FFMPEG_INCLUDE_PATH ${CMAKE_EXTERNAL_DEPS}/include/ffmpeg)
   if(MACPORTS)
     set(FFMPEG_INCLUDE_PATH /opt/local/include)
   else()
     set(FFMPEG_INCLUDE_PATH /usr/local/include)
   endif()
 
-  target_include_directories(${PLUGIN_NAME} SYSTEM PRIVATE ${FFMPEG_INCLUDE_PATH})
```

Comment:
This is needed when building Mac with MACPORTS currently (see line 63). Can we keep the line in?

Reply:
sure, my ax was a bit too sharpe

Reply:
fixed